### PR TITLE
add emscripten setup instruction

### PR DESCRIPTION
**Pull request by edap**

### CODE REVIEW

File: setup/emscripten/index.html.mako [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## -*- coding: utf-8 -*-
+<%inherit file="/_templates/markdown.mako" />
+
+emscripten setup
+===========
+
+Welcome to processing.js for Open Frameworks! Have you ever made a project in openFrameworks and been like "This is awesome! I want to put it on my online portfolio!". You could always make a video or take photos, but what about the UI aspect of your project? How can people see the awesome mouse effects you made? Enter Emscripten. Emscripten turns your C++ code into Javascript code in a matter of minutes! So you can put your awesome OF project "on the line" in your portfolio.
```
Comment (arturoc) [inline setup/emscripten/index.html.mako]: i would reword this, processing.js is written in javascript by the user while what we do is translate c++ to javascript so it might be confusing. also Open Frameworks -> openFrameworks

Comment (edap) [inline setup/emscripten/index.html.mako]: ok, fixed 99bca9449b4e8add85cc9c2d3d9639e185df0a1d

File: setup/emscripten/index.html.mako [hunk 2]
```diff
@@ -0,0 +1,55 @@
+## -*- coding: utf-8 -*-
+<%inherit file="/_templates/markdown.mako" />
+
+emscripten setup
+===========
+
+Have you ever made a project in openFrameworks and been like "This is awesome! I want to put it on my online portfolio!". You could always make a video or take photos, but what about the UI aspect of your project? How can people see the awesome mouse effects you made? Enter Emscripten. Emscripten translates your C++ code into Javascript code in a matter of minutes! So you can put your awesome OF project "on the line" in your portfolio.
+
+These instruction will show you how to compile one example project in the openFrameworks distribution, and can be reused whenever you want to translate an openFrameworks sketch to a web page. Assuming that you have downloaded openFrameworks 0.9 (it works only with OF versions >= 0.9) you only have to Install emscript and compile a project with it. Here the instruction steb by step.
```
Comment (arturoc) [inline setup/emscripten/index.html.mako]: it should be install emscripten

File: setup/emscripten/index.html.mako [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+If it does not work, it is probably because you are opening with safari, and safari does not support WebGL. Let's open it with another browser, like chrome
+
+```bash
+emrun --browser chrome bin/3DPrimitivesExample.html
+```
+If everything works, you should see this image
+
+![1](3dPrimitives.png)
+
+This Tutorial is token from this [blog post](http://www.reginafloresmir.com/blog/2015/5/14/openframeworks-on-the-line) by Regine Flores Mir.
```
Comment (arturoc) [inline setup/emscripten/index.html.mako]: there's a typo in token -> taken

Comment (arturoc) [inline setup/emscripten/index.html.mako]: also can you take the screenshot again? the latest version already exports a page with the OF logo instead of the emscripten one

**Comment by edap** [comment]
img changed and typo fixed

**Comment by arturoc** [comment]
thanks!!

**Comment by arturoc** [comment]
i'll add a link to it when i remake the download page to include the new palatforms and ides
